network-api/networkapi/buyersguide/templates/catalog.html

```diff
@@ -30,13 +30,28 @@ <h1 class="h1-heading text-center mt-0">{% trans "Be Smart. Shop Safe." %}</h1>
 
 {% block guts %}
 <div class="project-list-section">
-  <div class="creepiness-slider bg-white text-center">
+  <div id="sticky-bar" class="creepiness-slider bg-white text-center">
+
+    <!-- temporary location for dev work -->
+    <span id="product-filter-search" tabindex="0">
+      <span class="search-icon">&nbsp;</span>
+      <input type="text" role="searchbox" required id="product-filter-search-input" placeholder="Search product name..." class="body-large">
```

Comment:
This string needs to be wrapped for localization (and I replaced the three dots with the dedicated Unicode character `…` as it’s the convention)

```suggestion
      <input type="text" role="searchbox" required id="product-filter-search-input" placeholder="{% trans "Search product name…" %}" class="body-large">
```